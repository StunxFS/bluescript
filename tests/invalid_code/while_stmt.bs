fn main() {
    while 1 { }
}
