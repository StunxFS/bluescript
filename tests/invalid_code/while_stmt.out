tests/invalid_code/while_stmt.bs:2:11: error: non-boolean `while` condition
